healthcareai/common/top_factors.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression, LinearRegression
+
+from healthcareai.common.output_utilities import save_object_as_pickle
+
+
+def prepare_fit_model_for_factors(model_type, x_train, y_train):
+    """
+    Given a model type, train and test data
+    Args:
+        model_type:
+        x_train:
+        y_train:
+
+    Returns:
+        A fit model. Also saves it as a pickle file.
+    """
+
+    if model_type == 'classification':
+        algorithm = LogisticRegression()
+    elif model_type == 'regression':
+        algorithm = LinearRegression()
+    else:
+        algorithm = None
+
+    if algorithm is not None:
+        algorithm.fit(x_train, y_train)
+        save_object_as_pickle('factorlogit.pkl', algorithm)
+
+    return algorithm
+
+
+def find_top_three_factors(trained_model, x_test, debug=False):
+    """
+    Given a trained model and an x_test set, reverse engineer the top three feature importances
+    Args:
+        trained_model:
+        x_test:
+        debug:
+
+    Returns:
+        A tuple of the top three factors
+    """
+    # Populate X_test array of ordered column importance;
+    # Start by multiplying X_test values by coefficients
+    multiplied_factors = x_test.values * trained_model.coef_
+    feature_columns = x_test.columns.values
+
+    # initialize the empty factors
+    first_factor = []
+    second_factor = []
+    third_factor = []
+
+    # TODO: switch 2-d lists to numpy array
+    # (although would always convert back to list for ceODBC
```

Comment:
Can you change this comment to reference PyODBC (or whatever it is you're using) instead of ceODBC